.github/workflows/main.yml

```
@@ -54,14 +54,16 @@ jobs:
         - build_php_table_backend_utils
     runs-on: ubuntu-latest
     if: |
-        contains(fromJson('["success"]'), needs.build_datatypes.result)
+        always()
+        && (contains(fromJson('["success", "skipped"]'), needs.build_datatypes.result) || contains(fromJson('["success", "skipped"]'), needs.build_php_table_backend_utils.result))
```

Hele, nemělo by tady být takhle? 
```suggestion
        && (contains(fromJson('["success", "skipped"]'), needs.build_datatypes.result) && contains(fromJson('["success", "skipped"]'), needs.build_php_table_backend_utils.result))
```

---

Ha! diky jo ma to byt tak jak pises, upravil som to.

---

To byl ten důvod proč to fungujovalo i když nemělo